scaleway/resource_instance_security_group.go

```diff
@@ -126,16 +137,25 @@ func resourceScalewayInstanceSecurityGroupRead(d *schema.ResourceData, m interfa
 	_ = d.Set("inbound_default_policy", res.SecurityGroup.InboundDefaultPolicy.String())
 	_ = d.Set("outbound_default_policy", res.SecurityGroup.OutboundDefaultPolicy.String())
 
-	//
-	// Handle SecurityGroupRules
-	//
+	if d.Get("external_rules") != nil && d.Get("external_rules").(bool) {
```

Comment:
do you need to check `!= nil` here? 🤔

Reply:
Yes, I had to add it otherwise there was an error in a test. 

But maybe the right fix is to have a default value for `external_rules`. I don't know how that would work with `ConflictsWith`.

Reply:
Exactly, `false` should be the default value and it won't trigger `ConflictsWith `